### PR TITLE
Cleanup cluster waiting

### PR DESCRIPTION
**Pull request by luca3rd**
## What does this PR do?

This cleans up the cluster waiting UX.

- Hides retry-able errors from the user
  - showing these errors will only cause confusion for the user
- Increases time between cluster status requests 

Fixes #\<issue_number>

## Before submitting

- [ ] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [x] Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [x] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda

### CODE REVIEW

**Review by nicolai86 — APPROVED** [review]

**Review by ethanwharris — APPROVED** [review]

File: src/lightning_app/cli/cmd_clusters.py [hunk 1]
```diff
@@ -307,21 +307,24 @@ def _cluster_status_long(cluster: V1GetClusterResponse, desired_state: V1Cluster
...
 
-            WARNING: Any non-deleted cluster may be using resources.
-            To avoid incuring cost on your cloud provider, delete the cluster using the following command:
-                lightning delete cluster {cluster_id}
+                --------------------------------------------------------------
 
-            Contact support@lightning.ai for additional help
-            """
-        )
+                We are automatically retrying, and an automated alert has been created
```
Comment (tchaton) [inline src/lightning_app/cli/cmd_clusters.py:319]: Not very clear what the alert does.

**Review by tchaton — COMMENTED** [review]
LGTM !

**Review by tchaton — APPROVED** [review]
LGTM !

File: src/lightning_app/cli/cmd_clusters.py (hunk 1 above)
Comment (ethanwharris) [reply src/lightning_app/cli/cmd_clusters.py:319]: This existed before this PR so I guess we can leave for a follow-up 😃

**Pull request merged**